docs/howto/release-git-lfs.md

```diff
@@ -337,3 +337,269 @@ and then use that branch as the base for the PATCH release.
 
   5. Then follow the [guidelines](#building-a-release) above, using the
      `release-M.N` branch as the base for the new PATCH release.
+
+### Building security PATCH versions
+
+In our public security [policy](../../SECURITY.md) we request that
+potential vulnerabilities in the Git LFS client be reported to us
+via email.  Users sometimes also choose to open draft GitHub security
+advisories, although we do not encourage that option.
+
+If we determine that such a report is valid, we develop a PATCH release
+version of the client with a remediation for the security issue,
+following the steps below.
+
+  1. Open a new draft security advisory, fill out the relevant
+     details in the
+     [template](https://github.com/git-lfs/git-lfs/security/advisories/new),
+     and request a CVE identifier.
+
+  2. Create a temporary private fork of this repository from the draft
+     security advisory page, and use the fork to develop a resolution
+     of the vulnerability.
+
+  3. Create two PRs in the private fork, one to remediate the vulnerability
+     in the `main` development branch of the public `git-lfs/git-lfs`
+     repository, and one to do so in the appropriate `release-M.N` branch.
+
+     The PR which targets the release branch should have a final, extra
+     commit which adds an entry for the new PATCH release version in the
+     `CHANGELOG.md` file, describing the release and the security fix it
+     contains.  This commit should also include the changes generated
+     by the `script/update-version vM.N.P` command, as per the second step
+     in our standard release process.
+
+     If the `release-M.N` branch does not exist in the public repository,
+     create it as per our regular PATCH release process, and then create
+     the second PR in the private fork against the new branch.  Be prepared
+     to proceed relatively quickly, since the appearance of the new branch
+     serves as a public notice that we may be publishing a PATCH release
+     shortly.
+
+  4. From the draft security advisory page, use the "Merge pull requests"
+     option to merge both PRs simultaneously.
+
+     Note that both the release branch and the `main` branch in the private
+     fork must be up-to-date with the corresponding branches in the public
+     repository in order for the GitHub UI to determine whether the PRs have
+     no merge conflicts.
+
+  5. Publish the security advisory.
+
+  6. Follow our standard release process, starting with the step in which
+     we create a GPG-signed tag named `vM.N.P` on the merge commit to the
+     `release-M.N` branch.
+
+  7. After publication of the new release, update the `git-lfs.com` home
+     page with a banner message regarding the security PATCH release
+     by pushing a change to the `_includes/home/secondary.html` file
+     in the `git-lfs/git-lfs.github.com` repository.
```

Comment:
nit: Maybe add [this link](https://github.com/git-lfs/git-lfs.github.com/commit/3aaeafd3f729e2f2d696b7dd19e718b850222d34) as example? BTW: Do we always keep the announcement or do we remove it after some time?

Reply:
Good questions.  I've expanded this step's instructions a bit more fully and included an example of the banner message and formatting for it, which I think will help clarify things somewhat.